idaes/models_extra/temperature_swing_adsorption/fixed_bed_tsa0d.py

```diff
@@ -0,0 +1,4522 @@
+            if t != self.heating.time_domain.first():
+                if (
+                    iscale.get_scaling_factor(
+                        self.heating.component_mass_balance_ode[t, "CO2"])
+                        is None
+                ):
+                    iscale.set_scaling_factor(
+                        self.heating.component_mass_balance_ode[t, "CO2"],
+                        1e-5)
+                if (
+                    iscale.get_scaling_factor(
+                        self.heating.overall_mass_balance_ode[t]) is None
+                ):
+                    iscale.set_scaling_factor(
+                        self.heating.overall_mass_balance_ode[t], 1e-5)
+                if (
+                    iscale.get_scaling_factor(
+                        self.heating.energy_balance_ode[t]) is None
+                ):
+                    iscale.set_scaling_factor(
+                        self.heating.energy_balance_ode[t], 1e-6)
+
+        # scaling factors for cooling step
+        if iscale.get_scaling_factor(self.cooling.mole_frac) is None:
+            iscale.set_scaling_factor(self.cooling.mole_frac, 1e1)
+
+        if iscale.get_scaling_factor(self.cooling.temperature) is None:
+            iscale.set_scaling_factor(self.cooling.temperature, 1e-2)
+
+        if iscale.get_scaling_factor(self.cooling.pressure) is None:
+            iscale.set_scaling_factor(self.cooling.pressure, 1e-4)
+
+        if iscale.get_scaling_factor(self.cooling.temperature_dt) is None:
+            iscale.set_scaling_factor(self.cooling.temperature_dt, 1e-2)
+
+        if iscale.get_scaling_factor(self.cooling.pressure_dt) is None:
+            iscale.set_scaling_factor(self.cooling.pressure_dt, 1e-4)
+
+        for t in self.cooling.time_domain:
+            if t != self.cooling.time_domain.first():
+                if (
+                    iscale.get_scaling_factor(
+                        self.cooling.component_mass_balance_ode[t, "CO2"])
+                        is None
+                ):
+                    iscale.set_scaling_factor(
+                        self.cooling.component_mass_balance_ode[t, "CO2"],
+                        1e-5)
+                if (
+                    iscale.get_scaling_factor(
+                        self.cooling.overall_mass_balance_ode[t]) is None
+                ):
+                    iscale.set_scaling_factor(
+                        self.cooling.overall_mass_balance_ode[t], 1e-5)
+                if (
+                    iscale.get_scaling_factor(
+                        self.cooling.energy_balance_ode[t]) is None
+                ):
+                    iscale.set_scaling_factor(
+                        self.cooling.energy_balance_ode[t], 1e-6)
+
+        # scaling for inlet states
+        iscale.set_scaling_factor(self.flow_mol_in, 1e-3)
+        iscale.set_scaling_factor(self.temperature_in, 1e-2)
+        iscale.set_scaling_factor(self.pressure_in, 1e-5)
+
+    def plots(self):
```

Comment:
I would suggest moving this out to a separate utility method file, in part to help keep the size of the files down.

Reply:
also moved plots to the util file